lib/engine/game/g_18_ny/entities.rb

```diff
@@ -311,7 +311,7 @@ module Entities
             always_market_price: true,
             float_percent: 20,
             coordinates: 'J20',
-            color: '#ED1C24',
+            color: 'purple',
```

Comment:
Please use the hex code to stay consistent with the rest of the implementation. The purple you emailed me is #8C4776. Apply to the SVGs too.

Reply:
ok